packages/backend/src/peer/service.ts

```diff
@@ -283,3 +285,14 @@ async function getPeersPage(
     .getPage(pagination)
     .withGraphFetched('asset')
 }
+
+async function deletePeer(
+  deps: ServiceDependencies,
+  id: string
+): Promise<Peer | undefined> {
+  return Peer.query(deps.knex)
+    .withGraphFetched('asset')
+    .deleteById(id)
+    .returning('*')
+    .first()
```

Comment:
I like the pattern of returning the deleted item, similar to how it was done for `accessToken` [revokation](https://github.com/interledger/rafiki/pull/1073#discussion_r1101732342)